### PR TITLE
Chips Expansion

### PR DESCRIPTION
**Pull request by Greenjoe12345**
This adds a number of types of crisp flavours, along with multipack bags containing multiple packets of crisps.
The other types are not yet in vending machines, as I've heard about a rework of vending machines that will split out things a bit, and they could be added there.
The multipacks I would want to be added to the trader/warehouse/third party cargo hold loot, (along with stores on planetside places like Konyang when we are visiting those), as well as orderable from cargo? but that's config stuff so I can't touch that.
The flavours are:
Salted (red)
Cucumber (green)
Chicken (blue)
Dirtberry (brown)
and Phoron (actually shrimp cocktail, in a pink packet)
each of these has a multipack, and there is a special variety multipack containing salted, cucumber and chicken flavours.
image

### CODE REVIEW

**Comment by Greenjoe12345** [comment]
!review

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -525,15 +525,43 @@
 	bitesize = 2
 
 /obj/item/reagent_containers/food/snacks/chips
-	name = "chips"
-	desc = "Getmore potato chips. Not actually chips, these are instead flash-fried wafers of potato paste. Delicious!"
-	icon_state = "chips"
+	name = "\improper Getmore salted potato crisps"
+	desc = "Getmore potato crisps. Not actually crisps, these are instead flash-fried wafers of potato paste. Delicious!"
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:529]: Quick note: In American English, this is the definition of crisps.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
Please, just keep them called chips. This feels like a "changing them for the sake of changing them" They are already called chips in the files.

**Review by Generalcamo — CHANGES_REQUESTED** [review]
If you are going to rename them, change the paths too. Otherwise this will cause needless confusion for mappers, coders, and administrators.

**Comment by Greenjoe12345** [comment]
> Please, just keep them called chips. This feels like a "changing them for the sake of changing them" They are already called chips in the files.

As generalcamo said, they are seemingly actually crisps

> If you are going to rename them, change the paths too. Otherwise this will cause needless confusion for mappers, coders, and administrators.

The paths are far too much of a complex thing for me to change

**Comment by Greenjoe12345** [comment]
adjusting it now

**Review by DreamySkrell — APPROVED** [review]

**Comment by Greenjoe12345** [comment]
fixing this AGAIN. that fucking sorting pr

**Comment by Greenjoe12345** [comment]
a

**Comment by Greenjoe12345** [comment]
Fixed again.

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**